.travis.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
                 - echo "memory_limit=4096M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 
-                - composer self-update --1
+                - composer self-update --preview
```

Comment:
`composer self-update --2`?

Reply:
I kept preview to have the newest versions, sometimes not stable even. We had it before v2 was announced, but I don't have a strong opinion about it. But if we will change it, I would go with `--stable`